views/membership.erb

```diff
@@ -0,0 +1,261 @@
+<div class="hero">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-6">
+        <h1><a href="/clenstvo">Staňte sa naším <strong>členom</strong> a budte aj vy Slovensko.digital</a></h1>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="section main-content">
+  <div class="container">
+    <div class="row">
+      <div class="sidebar col-md-3">
+        <ul class="sidebar-menu list-unstyled">
+          <li><a href="/podpora">Podporte finančne</a></li>
+          <li><a href="/venujte-svoj-cas">Venujte svoj čas</a></li>
+          <li class="sidebar-menu-item-active">Členstvo</li>
+        </ul>
+      </div>
+      <div class="col-md-9">
+        <h1 class="content-heading text-center">Prečo byť členom?</h1>
+        <p class="lead"> “Prečo sa stať členom?” Lebo spolu, ako komunita, dosiahneme viac. Tvoríme alianciu firiem a jednotlivcov, ktorým záleží na lepších službách štátu a budúcnosti tejto krajiny. Sme hrdí na to, že podnikáme a pracujeme poctivo a spájame ľudí, ktorí to vidia rovnako.</p>
+
+        <p class="lead">Pridajte sa k nám a získate:</p>
+          <ul>
+            <li><p>Možnosť rozhodovať o budúcnosti Slovensko.Digital</p></li>
```

Comment:
Ja by som do tychto zoznamov tie `<p>` nedaval, lebo teraz to vyzera trocha prilis riedke:
![image](https://user-images.githubusercontent.com/1374745/31383731-6c280180-adbc-11e7-99d9-1ad57809b91f.png)

Tiez by to bolo fajn mat rovnako velkym pismom. Kludne od `Pridajte sa...` bez tej `lead` triedy.